packages/component_code_gen/helpers/langchain_helpers.py

```diff
@@ -60,23 +64,27 @@ def format_result(result):
     return result
 
 
+def get_llm():
+    if config['openai_api_type'] == "azure":
+        azure_config = config["azure"]
+        llm = AzureChatOpenAI(deployment_name=azure_config['deployment_name'],
```

Comment:
you can just `return AzureChatOpenAI...`, and do the same for the `ChatOpenAI` def below, then you can avoid the `llm = ` assignment and the final `return llm`